src/codegen/compvars.cpp

```diff
@@ -195,7 +216,8 @@ class InstanceMethodType : public ValuedCompilerType<RawInstanceMethod*> {
 
     int numFrameArgs() override { return obj_type->numFrameArgs() + function_type->numFrameArgs(); }
 };
-std::unordered_map<std::pair<CompilerType*, CompilerType*>, InstanceMethodType*> InstanceMethodType::made;
+std::unordered_map<std::pair<CompilerType*, std::pair<CompilerType*, CompilerType*>>, InstanceMethodType*>
```

Comment:
nor is it by default for std::pair :P  check out core/common.h if you're interested